src/cursor.js

```diff
@@ -138,6 +138,7 @@ export default class Cursor {
     this.range.insertNode(element)
     this.range.setStartAfter(preceedingElement)
     this.range.setEndAfter(preceedingElement)
+    this.host.normalize() // mend text nodes
```

Comment:
Is normalize safe to use at this point? Does it maintain the cursor position correctly?
E.g. if the inserted element has a text length of 1, the cursor might jump by 1 character.

Reply:
I tested it and it seemed to work as intended.